frontend/test/__support__/e2e/helpers/e2e-dashboard-helpers.js

```diff
@@ -21,6 +21,7 @@ export function editDashboard() {
 export function saveDashboard() {
   cy.findByText("Save").click();
   cy.findByText("You're editing this dashboard.").should("not.exist");
+  cy.wait(1); // this is stupid but necessary to due to the dashboard resizing and detaching elements
```

Comment:
@nemanjaglumac , I'm sure you will have thoughts on this, but at least in my local testing it allowed me to remove this block , which added several seconds to each test:
```js
        // this keeps the test from flaking because it's confused about the detached
        // edit-mode button
        cy.findByText(/^Edited a few seconds ago/).should("not.be.visible");
```

Reply:
the problem:
- all dashboard items get re-rendered and detached from the dom when you save a dashboard, so in the time between when cypress finds them and tries to click them, they're all new elements that it can no longer click

other things I tried
- waiting on data fetch does not work
- nemanja's solution was to wait for the edited text on the dashboard to fade away but that's on a 2 second timeout and is REALLY slow